boardswarm/README.md

```diff
@@ -318,6 +343,46 @@ devices:
           udev.ID_PATH: "pci-0000:00:14.0-usb-0:12.3"
 ```
 
+#### DFU Volumes
```

Comment:
This section will apply to any kind of volume (dfu, rockusb) not just dfu

Reply:
Generalised.